convex/stress.ts

```
@@ -0,0 +1,209 @@
+// ─── Core Formula ─────────────────────────────────────────────────────────────
+
+function calculateStress(metrics: {
+	overdueCount: number;
+	pendingSoon: number;
+	totalPending: number;
+	highPriorityPercent: number;
+}): { baseScore: number; finalScore: number; stressLevel: StressLevel; multiplierApplied: boolean } {
+	const { overdueCount, pendingSoon, totalPending, highPriorityPercent } = metrics;
+
+	// Step 1 – base formula
+	let baseScore = overdueCount * 10 + pendingSoon * 5 + totalPending * 2;
```

Biome (2.4.15)

[error] This let declares a variable that is only assigned once.

(lint/style/useConst)